core/controller/src/main/scala/whisk/core/controller/Actions.scala

```diff
@@ -336,40 +340,103 @@ trait WhiskActionsApi extends WhiskCollectionAPI {
         }
     }
 
+    /**
+     * helper method that fixes the sequence components that are in the default namespace
+     * replace default namespace witht the user namespace
+     */
+    private def fixDefaultPackage(seq: SequenceExec, user: Identity)(implicit transid: TransactionId): SequenceExec = {
+        // if components are part of the default namespace, they contain `_`; replace it!
+        val resolvedComponents = seq.components map { c: FullyQualifiedEntityName => FullyQualifiedEntityName(c.path.resolveNamespace(user.namespace), c.name) }
```

Comment:
Not needed to be there and doesn't add more clarity, while adding noise to me as a reader.